microcosm_metrics/decorators.py

```diff
@@ -29,10 +29,12 @@ def wrapper(*args, **kwargs):
                     return classifier(*args, **kwargs)
                 finally:
                     if classifier.label is not None:
+                        if tags is not None:
```

Comment:
It seems like you'd want to handle the `tags is None` case too and create a tag for the `classifier`, no?

Reply:
changed